keps/sig-network/3726-standard-application-protocols/README.md

```diff
@@ -176,6 +179,9 @@ type ServicePort struct {
   //
   // * Kubernetes-defined prefixed names:
   //   * 'kubernetes.io/h2c' - HTTP/2 over cleartext as described in https://www.rfc-editor.org/rfc/rfc7540
+  //   * 'kubernetes.io/ws'  - WebSocket over cleartext as described in https://www.rfc-editor.org/rfc/rfc6455
+  //   * 'kubernetes.io/wss' - WebSocket over TLS as described in https://www.rfc-editor.org/rfc/rfc6455
```

Comment:
nit: a bit confused why WS is added here when the description is about "raw"?

Reply:
See PR body - I was hoping the other PR would merge and then there wouldn't be a conflict in this one

Reply:
If that does happen, please rebase to avoid _any_ mention.

Reply:
done